### ISSUE TITLE
Error in loading the pretrained model

### ISSUE THREAD

**Issue by zhanglonghao1992** [facebookresearch/multiface#24]

**Comment by vexilligera** [comment]
Hi,

Sorry for the late reply. I tried
```
python -m torch.distributed.launch --nproc_per_node=1 test.py --data_dir dataset/m--20180227--0000--6795937--GHS --krt_dir dataset/m--20180227--0000--6795937--GHS/KRT --framelist_test dataset/m--20180227--0000--6795937--GHS/frame_list.txt --test_segment ./mini_test_segment.json --model_path pretrained_model/6795937_model.pth
```
and the model was correctly loaded. Could you provide the model name you loaded as well?

**Comment by wwpokenet** [comment]
I encountered the same issue by loading the pretrained model from the [link](https://fb-baas-f32eacb9-8abb-11eb-b2b8-4857dd089e15.s3.amazonaws.com/MugsyDataRelease/PretrainedModel/mini.pth) you provided in the `INSTALLATION.md`

**Comment by vexilligera** [comment]
> I encountered the same issue by loading the pretrained model from the [mini pretrained model](https://fb-baas-f32eacb9-8abb-11eb-b2b8-4857dd089e15.s3.amazonaws.com/MugsyDataRelease/PretrainedModel/mini.pth) you provided in the `INSTALLATION.md`

Hi, could you try the full model for now using the command previously mentioned? It's the same size as the mini model and it appears to work fine.
@cwuu Could you check if the cameras used for training the mini model are correct?

Thanks

**Comment by wwpokenet** [comment]
Yes I can load the full model. However the output textures look weird, like [these pred_tex](https://connecthkuhk-my.sharepoint.com/:f:/g/personal/qindafei_connect_hku_hk/Et3P6coybLlMgjW159I5VdcB5vH5i1D1hP8albHOqVFXQQ?e=Dkhf8X).

**Comment by vexilligera** [comment]
> Yes I can load the full model. However the output textures look weird, like [these pred_tex](https://connecthkuhk-my.sharepoint.com/:f:/g/personal/qindafei_connect_hku_hk/Et3P6coybLlMgjW159I5VdcB5vH5i1D1hP8albHOqVFXQQ?e=Dkhf8X).
> 
> The result.txt is:
> 
> ```
> Best screen loss 0.000000, best tex loss 0.070132,  best vert loss 0.002344, screen loss 0.000000, tex loss 0.070132, vert_loss 0.002344
> ```

Hi, this is because the model is conditioned on viewpoint. If the face part is occluded during training there's no supervision on that texture area, leading to these bright-light artifacts, which is expected.

**Comment by revanj** [comment]
Hi there! I was running the pretrained models and had something similar. Specifically, these model dimension mismatches happen for identities 002643814, 7889059, 5372021, 2183941, and 002914589. I hand wrote all the camera configs to include all cameras that exists in their respective folders, but for these entities, the pretrained models seems to be missing a large chunk of cameras (e.g. given 40 but requires 76, etc). identities 6795937 and 8870559 were able to run, howerver. Was I accidentally using a wrong network architecture?

Thanks

**Comment by cwuu** [comment]
Hi @revanj , 

We just update the codebase to include different camera configs for each identities and their pretrained-models with different architectures (w/o screen loss). Please let me know if it still doesn't work for your case. Thanks.

**Comment by avinabsaha** [comment]
Hi @cwuu , the mini model checkpoint is still having the size mismatch issue.
